docs/config.md

```diff
@@ -400,11 +400,14 @@ deleted if you push the branch with `jj git push --branch` or `jj git push
 
 # Alternative ways to specify configuration settings
 
-Instead of `~/.jjconfig.toml`, the config settings can be located at
-`$XDG_CONFIG_HOME/jj/config.toml` as per the [XDG specification]. It is an error
-for both of these files to exist.
-
-[XDG specification]: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+Instead of `~/.jjconfig.toml`, the config settings can be located under
+a platform-specific directory. It is an error for both of these files to exist.
+
+| Platform | Value                                  | Example                                                 |
+| :------- | :------------------------------------- | :------------------------------------------------------ |
+| Linux    | `$XDG_CONFIG_HOME`/jj                  | /home/alice/.config/jj/config.toml                      |
+| macOS    | `$HOME`/Library/Application Support/jj | /Users/Alice/Library/Application Support/jj/config.toml |
+| Windows  | `{FOLDERID_RoamingAppData}`\jj         | C:\Users\Alice\AppData\Roaming\jj\config.toml           |
```

Comment:
I would probably do
`$HOME/Library/Application Support/jj` | `/home/alice/.config/jj/config.toml`
and so on, rendered it does not look too pretty currently

Reply:
Yeah, I had just copied the style from the [upstream doc comments](https://github.com/dirs-dev/dirs-rs/blob/e99c8fc5667d11e027b934971913240e880e6b82/src/lib.rs#L153-L157), which only use inline code for variables, but I'd be fine using it for the entire path.

Reply:
I only put the directory in the **Value** column since there's [a TODO](https://github.com/martinvonz/jj/blob/a3973cc53f8a02fd644833c758190a3ca4272abb/src/config.rs#L195-L196) to possibly read all files in the config directory, and it was a bit shorter to only have the `config.toml` file name in the **Example** column.

Reply:
I pushed the fix for the inline code formatting to include the entire paths for both the **Value** and **Example** columns.

Reply:
hmm, sure, but it's not consistent with other rows in the table, that's what stood out, so by that logic other platforms should have no filename too.

And that todo is a todo for now, I think if/when that's implemented then this doc should reflect that, not now;
And if you remove the filename the only thing saying that it should be in fact `config.toml` (which is the current behavior) is the example.

Reply:
I believe it is consistent across rows currently, but it might look a bit confusing with the soft-wrapping on the diff? If you found the lack of `config.toml` under **Value* confusing, then I'm sure other people will too...I'll add it.

Reply:
oh wait, I am literally blind
yes, it was the wrapping, ugh, sorry

but yes the point stands about the doc reflecting the current state